src/utils/generate-output-json.ts

```diff
@@ -0,0 +1,81 @@
+const csvFilePath = "./public/data.csv";
+import csv from "csvtojson";
+import groupBy from "lodash.groupby";
+import fs from "fs";
+import relations from "../../public/output.json";
+
+function removeEmptyFields(obj: any) {
+  return Object.fromEntries(Object.entries(obj).filter(([_, v]) => v));
+}
+
+function parseTags(tags: any) {
+  const list = tags
+    .split(";")
+    .map((el: any) => el.trim())
+    .filter((item: any) => item)
+    .map((line: any) => {
+      const [key, value] = line.split(":");
+      return {
+        key: key.trim(),
+        value: value?.trim() || value,
+      };
+    });
+  const groupedList = groupBy(list, "key");
+
+  const output: any = {};
+
+  Object.keys(groupedList).forEach((key) => {
+    output[key] = groupedList[key].map((item: any) => item.value);
+  });
+  return output;
+}
+
+function cleanObject(obj: any): any {
+  if (Array.isArray(obj)) {
+    return obj.map(cleanObject).filter((v) => v !== undefined && v !== null);
+  } else if (obj !== null && typeof obj === "object") {
+    return Object.entries(obj).reduce((acc: any, [key, value]) => {
+      if (value === undefined) {
```

Comment:
use early return also here to make it easily understandable

Reply:
Fixed